### PR TITLE
modified README

### PR DESCRIPTION
**Pull request by Oleksii-White**
I added a header for the test.

### CODE REVIEW

**Comment by Oleksii-White** [comment]
@BiOksana is it ok ?

**Review by BiOksana — APPROVED** [review]
viewed

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # GitHubStudyRepo
-
+--------
+new changes)
```
Comment (rasnaut) [inline README.md:3]: No smile!

**Review by hakingdev — CHANGES_REQUESTED** [review]
remove ")"

**Review by Hellboy-Nishero — APPROVED** [review]

**Review by rasnaut — CHANGES_REQUESTED** [review]
Please change according my comments

**Comment by DeyanSpasovBerlin** [comment]
Back to begining!

**Review by hakingdev — APPROVED** [review]
Ok Ok

File: README.md (hunk 1 above)
Comment (Oleksii-White) [reply README.md]: ich habe das schöne gemacht

**Pull request merged**